README.rst

```diff
@@ -42,10 +42,45 @@ Run tests with Chrome::
 
     $ make test-dev
 
-Generate widgets pot file for plone translations::
+Generate widgets.pot file in the working directory for Plone translations::
 
     $ make i18n-dump
 
+If you are currently in the buildout.coredev/src/mockup folder and want to update
+the translations in plone.app.locales, first go back in the buildout.coredev
+folder, copy widgets.pot in the plone.app.locales package and resync the po files like
+this::
+
+    $ cd ../..
+    $ cp src/mockup/widgets.pot src/plone.app.locales/plone/app/locales/locales/widgets.pot
+    $ bin/buildout -c experimental/i18n.cfg  # to have the bin/i18n command
+    $ bin/i18n widgets
+
+If you did some changes in the js files and want to test them live in Plone:
+
+- go to Site Setup -> Resource Registries
+- click on the "Development Mode (only logged in users)" checkbox
+- click on the "Develop Javascript" button for the plone-editor-tools bundle
```

Comment:
plone-editor-tools isn't in Plone 5.1, "plone-logged-in" would be the equivalent there IIUC

Reply:
You're probably right. This PR is for mockup master so targeting Plone 5.2.x.